### ISSUE TITLE
Prieview broken

### ISSUE THREAD

**Issue by edvinsm** [vscode-restructuredtext/vscode-restructuredtext#44]
**Environment**
- [x] Extension version: 0.0.23
- [x] Visual Studio Code version: 1.12.1
- [x] Operating System version: Windows 10
- [x] Python version: 3.6.1
- [x] sphinx version: 1.5.5
- [ ] A sample of conf.py or its important fragments:

**Expected behavior**
show preview for an reStructuredText page
**Actual behavior**
Unable to open 'Preview 'filename.rst'':ErrorENOENT: no such file or directory
**Detailed Steps to reproduce the problem**
install extension
reopen code .
ctr+shift+r

preview(uri) function add last 3 characters from working directory name (documentation -> ion)
![preview-err](https://cloud.githubusercontent.com/assets/10815146/25996299/5e87e3d8-371f-11e7-9011-bd58c7efc711.png)

Quick fix: 
```diff
>         whole = whole.substring(3, ext) + ".html";
---
<         whole = whole.substring(0, ext) + ".html";

**Comment by lextm** [comment]
A quick fix is not enough. What exactly was the input of `uri`in your case?

**Comment by elpatron68** [comment]
We have the same problem here. After some investigation, I set up a fresh Sphinx directory with `sphinx-quickstart` Version 1.4.8 Python 3.6.0. Opening the preview of `index.rst` causes the same error message.

Applying ´.vscode/settings.json` doesn´t help either:

```
{
    "restructuredtext.builtDocumentationPath" : "_build/html",
    "restructuredtext.makefilePath" : ".",
    "restructuredtext.updateOnTextChanged" : "true"
}
```

I don´t know how to apply the quick fix...

**Comment by lextm** [comment]
![2017-05-17_08-40-06](https://cloud.githubusercontent.com/assets/425130/26154311/f4639efe-3adc-11e7-90e5-0dabe025f761.png)

If you create a new project like this, can you check if `index.rst` can be previewed correctly?

What's your choices during project creation?

**Comment by elpatron68** [comment]
Hi @lextm 
I tried it again today on a third (Windows 10) machine. I used default settings with `sphinx-quickstart`. I made a [screen cast](https://youtu.be/gThBzGN9hS0), same result.

**Comment by lextm** [comment]
Thanks, guys. Fixed in f5050fa6f2ca5930b4e7bedcc66816d9dee164ff

Please upgrade to 24.0.0 release.
